compiler/rustc_middle/src/mir/interpret/allocation.rs

```diff
@@ -358,10 +358,11 @@ impl Allocation {
     pub fn adjust_from_tcx<Prov: Provenance, Bytes: AllocBytes, Err>(
         &self,
         cx: &impl HasDataLayout,
+        mut alloc_bytes: impl FnMut(&[u8], Align) -> Bytes,
```

Comment:
```suggestion
        mut alloc_bytes: impl FnMut(&[u8], Align) -> Result<Bytes, Err>,
```
This lets you avoid the `unwrap` in Miri's `adjust_global_allocation`

Reply:
Implemented in a different commit. 👌